interface/main/tabs/js/tabs_view_model.js

```diff
@@ -129,7 +134,7 @@ function navigateTab(url,name,afterLoadFunction)
     }
     else
     {
-        curTab=new tabStatus(xl('New'),url,name,true,false,false);
+        curTab=new tabStatus("Loading <a style='font-size:80%'><i class='fa fa-spinner fa-pulse'></i></a>",url,name,label,true,false,false);
```

Comment:
translate Loading:
```
xl("Loading") + "<a style...
```

Reply:
`xl()` outputs undefined 😕 
![image](https://user-images.githubusercontent.com/47683556/76170571-25d05900-61a9-11ea-8b90-e37abc61a752.png)

Reply:
Probably not an issue in this line since previously New was being translated here. I am willing to bet it is in one of the other places where I recommended adding xl() use since guessing it is trying to call the xl() function before it was brought in by the library/js/utility.js script

Reply:
utility.js load takes longer now thats why were seeing. Only on first tab so I need to modify xl() loading for await or maybe just a promise. We can go ahead bring in and i'll fix up after I get with @im-Amitto for sortable.
@yashrajbothra  Just leave the xl() as is.